### PR TITLE
officiates use of Matroska Player and Matroska Reader

### PR DESCRIPTION
**Pull request by ablwr**
This PR will build on the definitions within https://github.com/Matroska-Org/matroska-specification/pull/198

### CODE REVIEW

File: notes.md [hunk 1]
```diff
@@ -104,23 +104,23 @@ file name   | `SegmentUID`                       | `PrevUID`
...
 
 Only one track of a kind MAY have its "default track" flag set in a segment. If a track entry does not contain the "default track" flag element then its default value "1" is to be used.
 
 ## Forced flag
 
-The "forced" flag tells the playback application that it MUST display/play this track or another track of the same kind that also has its "forced" flag set. When there are multiple "forced" tracks, the player SHOULD determined based upon the language of the forced flag or use the default flag if no track matches the use languages. Another track of the same kind without the "forced" flag may be use simultaneously with the "forced" track (like DVD subtitles for example).
+The "forced" flag tells the `Matroska Player` that it MUST display/play this track or another track of the same kind that also has its "forced" flag set. When there are multiple "forced" tracks, the `Matroska Player` SHOULD determined based upon the language of the forced flag or use the default flag if no track matches the use languages. Another track of the same kind without the "forced" flag may be use simultaneously with the "forced" track (like DVD subtitles for example).
```
Comment (robUx4) [inline notes.md]: s/determined/determine/ but all that proof reading can go in a separate patch.

Comment (ablwr) [reply notes.md]: I'm right here so I'll Girl Scout and fix it now 💪

File: menu.md [hunk 2]
```diff
@@ -5,31 +5,31 @@
...
+This document is a _draft of the Menu system_ that will be the default one in `Matroska`. As it will just be composed of a Control Track, it will be seen as a "codec" and could be replaced later by something else if needed.
 
 A menu is like what you see on DVDs, when you have some screens to select the audio format, subtitles or scene selection.
 
 ## Requirements
 
 What we'll try to have is a system that can do almost everything done on a DVD, or more, or better, or drop the unused features if necessary.
 
-As the name suggests, a Control Track is a track that can control the playback of the file and/or all the playback features. To make it as simple as possible for players, the Control Track will just give orders to the player and get the actions associated with the highlights/hotspots.
+As the name suggests, a Control Track is a track that can control the playback of the file and/or all the playback features. To make it as simple as possible for `Matroska players`, the Control Track will just give orders to the `Matroska Player` and get the actions associated with the highlights/hotspots.
```
Comment (mbunkus) [inline menu.md]: Capital `P` in the first change

Comment (ablwr) [reply menu.md]: squashed and pushed
